apps/cowswap-frontend/src/common/pure/ApproveButton/index.tsx

```diff
@@ -40,10 +40,15 @@ export function ApproveButton(props: ApproveButtonProps) {
       return (
         <>
           {/* we need to shorten this string on mobile */}
-          <span>
-            <Trans>
-              Allow CoW Swap to use your <TokenSymbol token={currency} />
-            </Trans>
+          <span
+            style={{
+              paddingLeft: '3px',
+              paddingRight: '3px',            
+             }}
+            >
+              <Trans>
+                Allow CoW Swap to use your <TokenSymbol token={currency} />
+              </Trans>             
           </span>
```

Comment:
This is something we shouldn't do.
It's adding inline styles which are not bet practice.

What I mean is that, we shouldn't use the `style={{}}` inside a component.
Instead, outside of this component, do something like:

```
const Wrapper = styled.span`
  padding-left: 3px;
  padding-righ: 3px; 
`
```

Or even combine all the paddings into a single row:

`padding: 0 3px;`

And here you replace `<span>` tags with the new `<Wrapper>` component.

For more expert input, I summon @fairlighteth

Reply:
I will give a try to fix it as you suggest. But I'd like to let you know that I just copied the way of adding a style that is used in the 80-90 lines in this file :)